lib/workers/branch/index.ts

```diff
@@ -333,6 +340,22 @@ export async function processBranch(
       const fileFilters = config.postUpgradeTasks.fileFilters || [];
 
       if (is.nonEmptyArray(commands)) {
+        // Persist updated files in file system so any executed commands can see them
+        for (const file of concat(
+          config.updatedPackageFiles,
+          config.updatedArtifacts
+        )) {
+          if (file.name !== '|delete|') {
+            let contents;
+            if (typeof file.contents === 'string') {
+              contents = Buffer.from(file.contents);
+            } else {
+              contents = file.contents;
+            }
+            await fs.outputFile(join(config.localDir, file.name), contents);
```

Comment:
I think this shouldn't be necessary for updated artifacts as they are always updated on disk by third party tools anyway. However, probably little harm either.

Reply:
Please use our util/fs library instead: https://github.com/renovatebot/renovate/blob/f97b19451539e5940258defdbc0afd4754df0be7/lib/util/fs.ts#L43

Reply:
I had the same thought about updated artifacts and I agree that there's probably little harm to include it. I've replaced `fs.outputFile` with `writeLocalFile`